<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="7574" started-at="2021-03-30T15:00:49 UTC" finished-at="2021-03-30T15:00:57 UTC">
    <groups>
    </groups>
    <test name="Test" duration-ms="7574" started-at="2021-03-30T15:00:49 UTC" finished-at="2021-03-30T15:00:57 UTC">
      <class name="com.guru99.tests.LoginTests">
        <test-method status="PASS" signature="preSetup()[pri:0, instance:com.guru99.tests.LoginTests@1c3a4799]" name="preSetup" is-config="true" duration-ms="182" started-at="2021-03-30T15:00:49 UTC" finished-at="2021-03-30T15:00:49 UTC">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preSetup -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.guru99.tests.LoginTests@1c3a4799]" name="setup" is-config="true" duration-ms="5633" started-at="2021-03-30T15:00:49 UTC" finished-at="2021-03-30T15:00:55 UTC">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="verifyUserLoginWithCorrectCridentials(java.lang.String, java.lang.String)[pri:0, instance:com.guru99.tests.LoginTests@1c3a4799]" name="verifyUserLoginWithCorrectCridentials" duration-ms="709" started-at="2021-03-30T15:00:55 UTC" finished-at="2021-03-30T15:00:56 UTC">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr317343]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[UmUvYga]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Guru99 Bank Manager HomePge] but found [Guru99 Bank Manager HomePage]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Guru99 Bank Manager HomePge] but found [Guru99 Bank Manager HomePage]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.guru99.tests.LoginTests.verifyUserLoginWithCorrectCridentials(LoginTests.java:26)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserLoginWithCorrectCridentials -->
        <test-method status="PASS" signature="postTestActions(org.testng.ITestResult)[pri:0, instance:com.guru99.tests.LoginTests@1c3a4799]" name="postTestActions" is-config="true" duration-ms="302" started-at="2021-03-30T15:00:56 UTC" finished-at="2021-03-30T15:00:56 UTC">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyUserLoginWithCorrectCridentials status=FAILURE method=LoginTests.verifyUserLoginWithCorrectCridentials(java.lang.String, java.lang.String)[pri:0, instance:com.guru99.tests.LoginTests@1c3a4799] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postTestActions -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.guru99.tests.LoginTests@1c3a4799]" name="tearDown" is-config="true" duration-ms="883" started-at="2021-03-30T15:00:56 UTC" finished-at="2021-03-30T15:00:57 UTC">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="postTeatDown()[pri:0, instance:com.guru99.tests.LoginTests@1c3a4799]" name="postTeatDown" is-config="true" duration-ms="530" started-at="2021-03-30T15:00:57 UTC" finished-at="2021-03-30T15:00:58 UTC">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postTeatDown -->
      </class> <!-- com.guru99.tests.LoginTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
